src/components/scroll-area.jsx

```diff
@@ -32,7 +32,7 @@ class ScrollArea extends React.Component {
         cursor: 'grab',
       };
       this.setState({ isScrolling: true, scrollAreaStyle: { style: cursorStyle } });
-      // event.preventDefault();
+      event.preventDefault();
```

Comment:
We should perhaps also consider applying `user-select: none` during drags: https://developer.mozilla.org/en-US/docs/Web/CSS/user-select